lib/matplotlib/tests/test_ticker.py

```diff
@@ -521,6 +521,12 @@ def test_sublabel(self):
         ax.set_xlim(0.5, 0.9)
         self._sub_labels(ax.xaxis, subs=np.arange(2, 10, dtype=int))
 
+    def test_LogFormatter_call(self):
+        # test _num_to_string method used in __call__
+        temp_lf = mticker.LogFormatter()
+        temp_lf.axis = FakeAxis()
+        assert str(temp_lf(10)) is not None
```

Comment:
Can we do a bit better and check that 1, 10, 100, 1000 are correctly rendered (`not None` is the example I used in the issue but is a pretty blunt test :) )